catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Association.js

```diff
@@ -142,15 +143,17 @@ define([
         },
         removeSegment: function (segment) {
```

Comment:
Yeah, that was more of a "for future consideration" comment. I agree that it's too big a change for this PR.